app/views/questionnaire.py

```diff
@@ -714,14 +714,21 @@ def _get_front_end_navigation(answer_store, current_location, metadata, routing_
     return None
 
 
-def get_page_title_for_location(schema, current_location):
+def _get_question_title_for_page(first_question, context):
+    if first_question.get('title') is not None:
```

Comment:
This might make more sense as `if 'title' in first_question`. My reasoning being that as a rule if `title` is defined on a schema then it should take precedence over `titles` - even if `title` is an empty string.

All this check does is adds implicit behaviour if somebody were to define `"title": null` (which validator wouldn't allow anyway).